### ISSUE TITLE
User can pan outside of map bounds

### ISSUE THREAD

**Issue by michaeltyson** [route-me/route-me#17]
There are upper and lower bounds to the map, nothing north of the north
pole. nothing south of the south pole.

RouteMe does not have any way to keep a user from scrolling much further
south than the south pole/north than the north pole.

I would love if there was a way to keep a user from panning beyond the
upper and lower bounds of the map, like maps.app does.

**Comment by amjaliks** [comment]
Isn't this fixed?

**Comment by michaeltyson** [comment]
Nope, not yet - I've started, but haven't finished it yet. The bounds limits code is there now, but it needs to set these limits in -[RMMapContents setTileSource].  

The latitudinal limits need to be > -90 and < 90 for the projection to work, though (slightly smaller than the full globe), presumably because it's only an approximation of a sphere (otherwise you get 'inf' returned for various functions), so gotta work out what the maximum bounds are and limit to that.

**Comment by cellininicholas** [comment]
I tried to fix this issue using the moveToLatLon method, but its not smooth :( I would very much like this feature to be fixed :)

**Comment by JaviSoto** [comment]
This also affects apps using offline maps. It'd be useful to prevent the user from scrolling beyond the bounds of the available tiles. Is there any way to accomplish this?
